.storybook/preview.ts

```diff
@@ -91,20 +91,12 @@ const preview: Preview = {
 }
 
 export const decorators: any = [
-  withThemeByClassName({
-    themes: {
-      light: "light",
-      dark: "dark",
-    },
-    defaultTheme: "light",
-  }),
   withThemeByDataAttribute({
     themes: {
-      light: "light",
-      dark: "dark",
+      "Isomer Classic": "isomer-classic",
+      "Isomer Next": "isomer-next",
     },
-    defaultTheme: "light",
-    attributeName: "data-mode",
```

Comment:
do we not need to specify `attributeName: "data-theme"`? or is that the default attributeName

Reply:
Yup that's [the default](https://github.com/storybookjs/storybook/blob/next/code/addons/themes/docs/api.md#withthemebydataattribute).